extensions/amp-consent/0.1/test/test-consent-state-manager.js

```diff
@@ -281,6 +281,18 @@ describes.realWin('ConsentStateManager', {amp: 1}, env => {
             'amp-consent:test',
             composeStoreValue(consentInfo)
           );
+
+          instance.update(CONSENT_ITEM_STATE.DISMISS);
+          yield macroTask();
+          storageSetSpy.resetHistory();
+          storageRemoveSpy.resetHistory();
+          expect(storageSetSpy).to.not.be.called;
+          expect(storageRemoveSpy).to.not.be.called;
+
```

Comment:
Might be overkill

Reply:
DISMISS and UNKNOWN can both be treated as an 'unknown' state. However 'DISMISS' will respect stored value while UNKNOWN doesn't. I think it's fine to add a check that `DIMISS` doesn't remove the stored value.

Reply:
Ahh I see. Sounds good.